app/src/main/java/com/keylesspalace/tusky/fragment/NotificationsFragment.java

```diff
@@ -863,6 +863,7 @@ private void onPreferenceChanged(String key) {
                     adapter.setMediaPreviewEnabled(enabled);
                     fullyRefresh();
                 }
+                break;
             }
             case "showNotificationsFilter": {
```

Comment:
While you're at it, it seems there is missing break statement too.

Reply:
Fixed.